### PR TITLE
new scheduled job to update confirmed cases once a day

### PR DESCRIPTION
**Pull request by dhachuel**
@MarcoBerlot let me know what you think

### CODE REVIEW

File: opendemic/scheduler/confirmed_cases.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	data['longitude'] = pd.to_numeric(data['longitude'])
+	data['biological_sex'] = data['sex'].apply(lambda x: 'F' if x == 'female' else ('M' if x == 'male' else np.nan))
+	data['symptom'] = 'verified_covid19'
+	data['source'] = 'beoutbreakprepared'
+	data['source_id'] = data['ID']
+	data['id'] = data['source_id'].apply(lambda x : str(uuid.uuid4()))
+	data['human_id'] = data['id']
+	try:
+		source_ids_to_remove = get_existing_source_ids()
+	except Exception as e:
```
Comment (MarcoBerlot) [inline opendemic/scheduler/confirmed_cases.py:57]: Not sure I see the point of catching an exception, to then raise it again. I believe that if you don't catch if you get the same result.

File: opendemic/scheduler/confirmed_cases.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import requests
+import sqlalchemy
+from sqlalchemy.engine.base import Engine
+from io import BytesIO
+from typing import List
+
+_CONFIRMED_CASE_DATA_URL: str = "https://github.com/beoutbreakprepared/nCoV2019/blob/master/latest_data/latestdata.tar.gz?raw=true"
+
+
+def generate_db_uri():
```
Comment (MarcoBerlot) [inline opendemic/scheduler/confirmed_cases.py:17]: Would it be possible to have this function in one place in the single app and keep referencing to it?

File: opendemic/scheduler/confirmed_cases.py (hunk 1 above)
Comment (dhachuel) [reply opendemic/scheduler/confirmed_cases.py:57]: That was intended as a placeholder. Forgot to add the logger there. Good catch

File: opendemic/scheduler/confirmed_cases.py (hunk 2 above)
Comment (dhachuel) [reply opendemic/scheduler/confirmed_cases.py:17]: Good point. I'll take care of that.

**Pull request merged**